src/connectivity.jl

```diff
@@ -83,3 +83,32 @@ function strongly_connected_components(g::DiGraph)
     end
     return components
 end
+
+# Computes the (common) period for all nodes in a strongly connected graph
+function strongly_connected_period(g::DiGraph)
```

Comment:
In short I'm undecided. My reason for not exporting this is that `strongly_connected_periods` returns a single integer, and `periods` returns a list of int's, one for each node. I'm happy to export both incase the user knows whether the graph will be strongly connected ahead of time.